<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickClipboardWindow</class>
 <widget class="QDialog" name="QuickClipboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>105</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>105</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>105</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuickClipboardWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>New filename</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="typeComboBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="memeTypeLabel">
       <property name="text">
        <string>Meme type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="fileNameInput"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>fileNameInput</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
